apps/web/src/components/submission/components/Credential.tsx

```diff
@@ -1,6 +1,5 @@
 import { useEffect, useRef } from 'react';
```

Comment:
Any particular reason why the file permissions needed to be changed on this file?

Reply:
I am not sure what you mean here. What file permissions were changed?

Reply:
he means `100644 → 100755`, shows a change in file permissions.  Did you run `chmod` locally?

Reply:
Ohh, I have no idea how that happened. Let me take a look